### PR TITLE
Added detailed hydra description with OWL vocabulary

### PR DESCRIPTION
**Pull request by alien-mcl**
This pull request comes with a minor _extension_ that adds detailed hydra description created in OWL.

Feel free to deliberate more on those descriptions.

### CODE REVIEW

File: hydra-in-owl/hydra_in_owl.ttl [hunk 1]
```diff
@@ -0,0 +1,108 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+@prefix hydra: <http://www.w3.org/ns/hydra/core#>.
+@prefix hiowl: <http://www.w3.org/ns/hydra/in-owl#>.
```
Comment (tpluscode) [inline hydra-in-owl/hydra_in_owl.ttl]: What do you think about using `w3id.org` for all extensions as proposed in #7?

I guess that in this case that would not be in the same namespace as all the rest

File: hydra-in-owl/index.bs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     	"rdfs": {
+     	    "authors": [
+     	        "Dan Brickley",
+     	        "R.V. Guha",
+     	        "Brian McBride"
+     	    ],
+     	    "href": "https://www.w3.org/TR/rdf-schema/",
+     	    "title": "Resource Description Framework Schema",
+     	    "status": "SD",
+     	    "publisher": "W3C"
```
Comment (tpluscode) [inline hydra-in-owl/index.bs]:
```suggestion
```

These should be unnecessary. Bikeshed maintains a list of most common standard specification which will link automatically to references

File: hydra-in-shacl/index.bs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+     	"rdfs": {
+     	    "authors": [
+     	        "Dan Brickley",
+     	        "R.V. Guha",
+     	        "Brian McBride"
+     	    ],
+     	    "href": "https://www.w3.org/TR/rdf-schema/",
+     	    "title": "Resource Description Framework Schema",
+     	    "status": "SD",
+     	    "publisher": "W3C"
```
Comment (tpluscode) [inline hydra-in-shacl/index.bs]: Ditto
```suggestion
```

**Review by tpluscode — CHANGES_REQUESTED** [review]
I don't like it. It looks like you simply search/replaced the OWL to become SHACL, but shapes would be different, used differently, and serve different purpose than OWL constraints.

I propose we even take a step back there and go class-by-class. As is, the document is a little large to try and review in a meaningful manner. My proposal would be begin with `hydra:ApiDocumentation`'s shape first and follow the vocabulary to add the rest in subsequent PRs

**Comment by alien-mcl** [comment]
No - it was not the case. Both OWL and SHACL versions were written from scratch separately, but were put against each other to have i.e. cardinalities in sync.

My entry point was a Hydra rewritten to turtle notation (I'll make a separate PR for that in Specification repo) and then class by class, predicate by predicate both OWL restrictions and SHACL shapes were created. It may look similar though as the order of the terms is the same (due to fact of using same entry point).

I also totaly disagree to move in a PR per term basis - it'll take ages while the task is quite simple. I may consider separating OWL and SHACL though.

**Comment by alien-mcl** [comment]
BTW, is there any list of valid statuses bikeshed accepts? The spec does not enumerate any and existing specifications are not using two letter derivatives.

**Comment by tpluscode** [comment]
It is not a simple task, as evidenced by this PR. I cannot really comment on the OWL but there is much to improve in SHACL and it will be much more productive to start small and progress gradually 

I think you're looking for https://tabatkins.github.io/bikeshed/#metadata-status?

**Comment by alien-mcl** [comment]
Hmm - I think it's as simple as creating old school pseudo classes, but not with a programming language but with other means. Let us not over-engineer it.

I see that I'll have to make OWL and SHACL separated to make it simpler.

**Comment by alien-mcl** [comment]
I've moved SHACL to separate PR leaving OWL alone.

File: hydra-in-owl/hydra_in_owl.ttl (hunk 1 above)
Comment (alien-mcl) [reply hydra-in-owl/hydra_in_owl.ttl]: I've switch to w3id.

File: hydra-in-owl/index.bs (hunk 2 above)
Comment (alien-mcl) [reply hydra-in-owl/index.bs]: Removed

**Comment by tpluscode** [comment]
The changes got lost here too 🤔

**Comment by alien-mcl** [comment]
Ok - I've recovered all changed files as these were ... ignored :/

**Review by tpluscode — APPROVED** [review]
I think I should "tentatively" approve this. As I said, I feel that this is too big and it's hard to review and verify correctness.

That said I do not intend to actually work on the OWL representation anyway in the foreseeable future...

**Pull request merged**